web/src/components/eventslist-component/EventFilters.js

```diff
@@ -0,0 +1,77 @@
+import React, {Component} from 'react';
+import Cards from './EventCards';
+import {
+  ToggleButton,
+  ToggleButtonGroup,
+  Jumbotron
+} from 'react-bootstrap';
+
+
+class EventFilters extends Component {
+  constructor(props) {
+    // get area selection from props
+    super(props);
+
+    // save the arrays of filter names and filter toggle(selected or not) in state
+    // Todo: mapping filterNames string-integer(enum) after db builiding complete.
+    const filterNames = ['BookClub', 'Movie', 'Wine', 'SciFi', 'Sport'];
+    const filterToggles = [];
+
+    for (let j = 0; j < filterNames.length; j++)
+      filterToggles.push(false);
+
+    this.state = {
+      filterNames: filterNames,
+      filterToggles: filterToggles,
+    };
+  }
+
+  // Function for update filter toggled value to new value
+  toggleHandler = val => {
+    console.log('toggle:', val);
+
+    const updatedFilterToggles = [];
+    for (let j = 0; j < this.state.filterToggles.length; j++)
```

Comment:
Please go through all for loops and try to update the variable names to more meaningful ones